### DISCUSSION TITLE
Question for cite/reference hyperlink

### DISCUSSION THREAD

**Discussion by wisejky** [nmfs-opensci/quarto-thesis#24] [Q&A] [answered]
Hallo！！

I'm encountering an issue with citations in my document. While the references appear correctly both in the body text and the reference section, it seems that the **hyperlink** functionality from the body text to the references **is not working**. How can I troubleshoot and resolve this problem?

`format:
  quarto-thesis-pdf: 
    link-citations: true`
after setting  link-citations as true
![image](https://github.com/nmfs-opensci/quarto-thesis/assets/120883966/49d9789c-86fa-4198-a9e2-dfcce12b3fd0)

before is ：
![image](https://github.com/nmfs-opensci/quarto-thesis/assets/120883966/ded92a54-0ce4-4626-998b-7352d80ebb03)

How can I make hyperlinks work? Thank you everyone!

**Comment by eeholmes** [comment]
Is this related to this bug? https://github.com/nmfs-opensci/quarto-thesis/issues/21

**Reply by wisejky** [reply, marked as answer]
[quarto-dev/quarto-cli/discussions/9238#discussion-6442753](https://github.com/quarto-dev/quarto-cli/discussions/9238#discussion-6442753)

i just find it can fix my citation problem.

tomorrow i would check it again about the order of several tex files

thanks for your template！！！ it is helpful
